### ISSUE TITLE
some image is loaded upside down

### ISSUE THREAD

**Issue by zlw21gxy** [aimerykong/predictive-filter-flow#4]
![image](https://user-images.githubusercontent.com/46110602/56873922-9296a500-6a68-11e9-9520-4f1d3af9fec5.png)

**Comment by zlw21gxy** [comment]
I wonder why we need do it?Flip image is some kind like data argument?
`        if self.set_name=='train' and np.random.random(1)>0.5:
            image1 = image1.transpose(PIL.Image.FLIP_TOP_BOTTOM)
            image2 = image2.transpose(PIL.Image.FLIP_TOP_BOTTOM)
        if self.set_name=='train' and np.random.random(1)>0.5:
            image1 = image1.transpose(PIL.Image.FLIP_LEFT_RIGHT)
            image2 = image2.transpose(PIL.Image.FLIP_LEFT_RIGHT)`

**Comment by aimerykong** [comment]
Yes. It is for augmenting training data. As mgPFF is a low-level vision
method, meaning that it does not need to understand high-level semantics,
such flip and flop augmentation can be applied here. But it is optional
when you have enough training data.
